microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+
+See also:
+
+* [Edge Developer Tools for Visual Studio (Preview)](https://devblogs.microsoft.com/visualstudio/?p=237066&preview=1&_ppp=7aa7aef54f), at Visual Studio DevBlogs.
+* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio) in _Visual Studio for web development_.
+
+
+<!-- ====================================================================== -->
+## Layers in 3D View
+
+<!-- Title: Layers in 3D View -->
+<!-- Subtitle: The Layers tool isn't going away - find it in the 3D View tool today. -->
+
+We previously added an improved version of Layers in the **3D View** tool, and we're removing the stand-alone version (the **Layers** tool) after users have enough time to make the transition. The **Layers** tool will display a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool there.
+
+In the following image, the **Focus Mode** UI is shown.  **Focus Mode** has a **Quick View** area at the bottom, in place of the **Drawer**.  The **Layers** tool, which is shown open in **Quick View**, contains only a link, to open the **3D View** tool above it:
+
+![The Layers tool now links to the 3D View tool with the Composited Layers view open.](../../media/2022/03/layers-3d-view-tool.png)
+<!-- work item > layersmove.gif -->
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
+
+
+<!-- ====================================================================== -->
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
+
+Starting in Microsoft Edge version 99, you can unminify a call stack in a performance profile, as follows:
+1. After recording a profile, in the **Performance** tool, click the new **Unminify** button.
+1. Download the unminified performance profile.<!--how?-->
+1. Import the unminified performance profile into the **Performance** tool.<!--how?-->
+
+The **Unminify** button will use source maps, provided that they are hosted side-by-side with your production code, to unminify the call stack in the **Performance** tool.
+
+The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, and **bn**:
```

Comment:
What do you think about doing these 2 images side-by-side? Are we not doing side-by-side images any more? I feel like in this case, it might make it clearer what the changes are to the call stack but I'll defer to you

Reply:
https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#columns - "While the columns extension still works, we no longer recommend it for creating custom layouts. We've found that many custom column layouts have accessibility issues or otherwise violate Docs style guidelines. Don't create custom layouts. Use standard Docs features."

Side-by-side image presentation was implemented by :::image::: custom markup tagging.  To make the articles legible/readable by simply scrolling, instead of mode-switching to having to pop-out / zoom the images, and as part of the Initiative: _Improve the content authoring / Content Developer experience_, we switched from ::: custom markup tagging to standard Markdown tagging, and no longer use ::: column tagging or ::: image tagging.

Now we present content by figuring out ways to leverage simple standard Markdown as much as possible, rather than relying on custom markup and column layout and image taging.  eg one benefit of this is color syntax coloring of alt-text while authoring in .md files in VS Code, producing better alt-text.